Libraries/Opc.Ua.Security.Certificates/Opc.Ua.Security.Certificates.csproj

```diff
@@ -46,6 +46,13 @@
     <PackageReference Include="Portable.BouncyCastle" Version="1.8.10" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Please remove the package reference to Nerdbank.Gitversioning. It is included in version.props for all projects and tied to an older version which isknown working with our 4 digit versioning.